pkg/extprom/http/metrics.go

```diff
@@ -29,26 +30,37 @@ func newDefaultMetrics(reg prometheus.Registerer, buckets []float64, extraLabels
 			},
 			append([]string{"code", "handler", "method"}, extraLabels...),
 		),
+
 		requestSize: promauto.With(reg).NewSummaryVec(
 			prometheus.SummaryOpts{
 				Name: "http_request_size_bytes",
 				Help: "Tracks the size of HTTP requests.",
 			},
 			append([]string{"code", "handler", "method"}, extraLabels...),
 		),
+
 		requestsTotal: promauto.With(reg).NewCounterVec(
 			prometheus.CounterOpts{
 				Name: "http_requests_total",
 				Help: "Tracks the number of HTTP requests.",
 			},
 			append([]string{"code", "handler", "method"}, extraLabels...),
 		),
+
 		responseSize: promauto.With(reg).NewSummaryVec(
 			prometheus.SummaryOpts{
 				Name: "http_response_size_bytes",
 				Help: "Tracks the size of HTTP responses.",
 			},
 			append([]string{"code", "handler", "method"}, extraLabels...),
 		),
+
+		inflightHTTPRequests: promauto.With(reg).NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: "http_inflight_requests",
+				Help: "Current number of HTTP requests the handler is responding to.",
+			},
+			append([]string{"handler"}, extraLabels...),
```

Comment:
All of the other metrics also consider a `method` label. Is there any reason not to collect that label here as well?